packages/components/src/Description/Value/styles.ts

```diff
@@ -6,11 +6,20 @@ import { Typography } from '../../Typography';
 
 import { type ValueProps } from './Value';
 
-export const StyledTypography = styled(Typography)`
-  cursor: pointer;
+type StyledTypographyProps = {
+  $canCopy: ValueProps['canCopy'];
```

Comment:
```suggestion
  $canCopy?: boolean;
```
https://kaluga-astral.github.io/style-guide/docs/rules/react/styled#%D1%82%D0%B8%D0%BF%D1%8B-%D0%B4%D0%BB%D1%8F-props-styled-%D0%BA%D0%BE%D0%BC%D0%BF%D0%BE%D0%BD%D0%B5%D0%BD%D1%82%D0%B0-%D0%BD%D0%B5-%D0%B4%D0%BE%D0%BB%D0%B6%D0%BD%D1%8B-%D0%B7%D0%B0%D0%B2%D0%B8%D1%81%D0%B5%D1%82%D1%8C-%D0%BE%D1%82-props-%D0%BA%D0%BE%D1%80%D0%BD%D0%B5%D0%B2%D0%BE%D0%B3%D0%BE-%D0%BA%D0%BE%D0%BC%D0%BF%D0%BE%D0%BD%D0%B5%D0%BD%D1%82%D0%B0